misc/snapshotter/snapshotter.sh

```diff
@@ -97,9 +104,53 @@ function options_handler() {
 
 }
 
+function configure_snapshotter() {
+
+    if [ "${CONTAINER_RUNTIME}" != "containerd" ]; then
+        die "not supported container runtime: ${CONTAINER_RUNTIME}"
+    fi
+
+    # Copy the container runtime config to a backup
+    cp "$CONTAINER_RUNTIME_CONFIG" "$CONTAINER_RUNTIME_CONFIG".bak.nydus
+
+    # Check and add nydus proxy plugin in the config
+    if toml check $CONTAINER_RUNTIME_CONFIG "proxy_plugins" >/dev/null; then
+        if [ "$(toml get $CONTAINER_RUNTIME_CONFIG proxy_plugins | jq length)" == "0" ]; then
+            echo "there is no proxy plugin!"
+            cat <<EOF >>"$CONTAINER_RUNTIME_CONFIG"
```

Comment:
is it possible to add the proxy_plugins entire section with `toml` instead of `cat`?

Maybe have a variable to hold `/run/containerd-nydus/containerd-nydus-grpc.sock`?

Reply:
>Maybe have a variable to hold /run/containerd-nydus/containerd-nydus-grpc.sock?
Fixed.

Reply:
> is it possible to add the proxy_plugins entire section with toml instead of cat?

I don't think that's possible due to a limitation on the `toml-cli`.